drivers/sensor/lsm6dsl/lsm6dsl.h

```diff
@@ -0,0 +1,712 @@
+/* sensor_lsm6dsl.h - header file for LSM6DSL accelerometer, gyroscope and
+ * temperature sensor driver
+ */
+
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __SENSOR_LSM6DSL_H__
+#define __SENSOR_LSM6DSL_H__
+
+#include <zephyr/types.h>
+#include <i2c.h>
+#include <misc/util.h>
+
+
+#define LSM6DSL_REG_FUNC_CFG_ACCESS			0x01
+#define LSM6DSL_MASK_FUNC_CFG_EN			BIT(7)
+#define LSM6DSL_SHIFT_FUNC_CFG_EN			7
+#define LSM6DSL_MASK_FUNC_CFG_EN_B			BIT(5)
+#define LSM6DSL_SHIFT_FUNC_CFG_EN_B			5
+
+#define LSM6DSL_REG_SENSOR_SYNC_TIME_FRAME		0x04
+#define LSM6DSL_MASK_SENSOR_SYNC_TIME_FRAME_TPH		(BIT(3) | BIT(2) | \
```

Comment:
Alignment looks ok to me in this file if you apply the patch. I think github displays it this way because each line has an extra + at the beginning.